cfbs/masterfiles/generate_release_information.py

```diff
@@ -7,16 +8,23 @@
 DOWNLOAD_PATH = "downloaded_masterfiles"
 
 
-def generate_release_information(omit_download=False, check=False):
+def generate_release_information(omit_download=False, check=False, min_version=None):
     if not omit_download:
         print("Downloading masterfiles...")
 
-        downloaded_versions = download_all_versions(DOWNLOAD_PATH)
+        downloaded_versions = download_all_versions(DOWNLOAD_PATH, min_version)
 
         print("Download finished. Every reported checksum matches.")
     else:
         downloaded_versions = immediate_subdirectories(DOWNLOAD_PATH)
 
+        downloaded_versions = list(
```

Comment:
What happens if you omit the `list()` function? I suspect it might work just fine.

Reply:
I think it's better to keep it so that things like `print` work as expected without modifications.